larq_compute_engine/tflite/kernels/bconv2d_params.h

```diff
@@ -11,27 +11,26 @@ namespace bconv2d {
 
 const int kTensorNotAllocated = -1;
 
-typedef struct {
-  // filters tensor dimensions
+struct TfLiteBConv2DParams {
+  // Filters tensor dimensions
   std::int32_t filter_width{0};
   std::int32_t filter_height{0};
   std::int32_t channels_in{0};
   std::int32_t channels_out{0};
 
-  // strides
+  // Strides
   std::int32_t stride_height{0};
   std::int32_t stride_width{0};
 
-  // dilations
+  // Dilations
   std::int32_t dilation_height_factor{0};
   std::int32_t dilation_width_factor{0};
 
-  // padding
+  // _adding
   TfLitePadding padding_type{};
   TfLitePaddingValues padding_values{};
   int pad_value = 0;  // Must be 0 or 1
 
-  TfLiteFusedActivation fused_activation_function = kTfLiteActNone;
```

Comment:
This value was never actually being read from - the relevant info is the clamp min/max that is stored just below.